src/psyclone/tests/test_files/gocean1p0/kernel_invalid_fortran.f90

```diff
@@ -38,7 +38,7 @@ subroutine compute_cu_code(i, j, cu, p, u)
     real(wp), intent(out), dimension(:,:) :: cu
     real(wp), intent(in),  dimension(:,:) :: p, u
 
-    CU(I,J) = 0.5d0*(P(i+1,J)+P(I,J))*U(I,J)
+    GO_CU(I,J) = 0.5d0*(P(i+1,J)+P(I,J))*U(I,J)
```

Comment:
This should not have the GO_ suffix.

Reply:
My apologies, I didn't know. I used it to keep track of what I had done ... and that obviously wasn't exactly too successful 😏 It seems that I somehow missed that file in my makefile, and so it was never compiled and tested *sigh* OK, I fixed that now - thanks!

Reply:
That's OK - how were you to know if we didn't tell you? :-) Thanks for fixing.